Autom/GLA_BSA-5_BSA-1

LanderoX144 opened the pull request:
…cion de estos datos en edit

AlanDiCha123 committed:
Resolvi conflictos entre dev y autom/parameterizacion

AlanDiCha123 committed:
All fields automated in form

AlanDiCha123 committed:
Automated testing of all fields fixed

AlanDiCha123 committed:
Automated testing of all fields fixed with parametrize

AlanDiCha123 committed:
Automated testing of expected fields on EditCustomer

Landero144 committed:
Hice la automatizacion del añadir un user nuevo, me falta la verificacion de estos datos en edit

Landero144 committed:
Hice la automatizacion del añadir un user nuevo, me falta la verificacion de estos datos en edit

AlanDiCha123 committed:
Random email generator added

AlanDiCha123 committed:
Added more time for jenkins

eduxsan commented on .vscode/launch.json:
```diff
@@ -11,7 +11,7 @@
       "args": [
         "-k",
-        "test_metod_name",
+        "test_BSA_1_GLA",
```
Please revert this change

eduxsan commented on Tests/New_Customer/test_new_customer.py:
```diff
@@ -23,3 +23,33 @@ def test_new_customer_input_message(self):
+        addCustomer.write_on_input("email","Hola@test.com")
+        addCustomer.click_on_input("Submit")
+        
+
+
+        
+        
+        
+
```
Please remove this empty lines

AlanDiCha123 committed:
Addressed requested changes

Landero144 committed:
Ya termine la verificacion

Landero144 committed:
Ya termine la verificacion

LanderoX144 committed:
Autom/AC BSA 3 BSA 1 (#12)

# This branch includes all the automated tesings for all fields with its
expecting data
I also would include the automation with Jenkins, but unfortunately it's
turned off :(

Landero144 committed:
Merge branch 'autom/GLA_BSA-5_BSA-1' of github.com:FESAr-SQA/SQA101 into autom/GLA_BSA-5_BSA-1

eduxsan commented on Pages/new_customer_page.py:
```diff
@@ -33,3 +61,161 @@ def verify_message_by_input_name(self, input_name, expected_message):
+                element = self.driver.find_element(*NewCustomerLocators.customer_address_input)
+        except Exception as e: 
+                pass
+    def verify_message_by_input_name(self, input_name, expected_message):
+        pass
+        
+
```
Please remove this

eduxsan commented on Tests/New_Customer/test_new_customer.py:
```diff
@@ -20,6 +23,157 @@ def test_new_customer_input_message(self):
+    
+        
+        
+        
```
Please remov ethis

Landero144 committed:
Amarillo los platanos

Landero144 committed:
awa de uwu

AlanDiCha123 commented on Locators/locators.py:
```diff
@@ -16,8 +16,52 @@ class LoginLocators:
+        customer_telephone_input = (By.XPATH, "/html/body/table/tbody/tr/td/table/tbody/tr[11]/td[2]/input")
+        customer_email_input = (By.XPATH, "/html/body/table/tbody/tr/td/table/tbody/tr[12]/td[2]/input")
+        submit_add_customer_button = (By.XPATH, "/html/body/table/tbody/tr/td/table/tbody/tr[13]/td[2]/input[1]")
+        id_customer_reg_msg = (By.XPATH, '//*[@id="customer"]/tbody/tr[4]/td[2]')
```
Check your identation

AlanDiCha123 commented on Pages/new_customer_page.py:
```diff
@@ -14,16 +15,43 @@ def click_on_input(self, input_name):
                 element = self.driver.find_element(*NewCustomerLocators.customer_name_txt)
```
Where is your "if" statement?

AlanDiCha123 replied:
Sorry

AlanDiCha123 commented on Pages/new_customer_page.py:
```diff
@@ -14,16 +15,43 @@ def click_on_input(self, input_name):
+            print(e)
 
     def verify_message_by_input_name(self, input_name, expected_message):
         try:
```
The "except" is missing

AlanDiCha123 replied:
Sorry

AlanDiCha123 commented on Tests/New_Customer/test_new_customer.py:
```diff
@@ -1,14 +1,17 @@
 from Pages.new_customer_page import NewCustomerPage
 import pytest
 import time
+from Pages.edit_customer import EditCustomerPage
```
Import is used twice

AlanDiCha123 replied:
Sorry

AlanDiCha123 reviewed (changes requested):
Check your code

Landero144 committed:
FINAAAAL

AlanDiCha123 reviewed (approved):
Well done
